### PR TITLE
Sonic arm64 changes

### PR DESCRIPTION
**Pull request by antony-rheneus**
Marvell arm64 changes over sonic

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: Makefile.work [hunk 1]
```diff
@@ -126,7 +126,7 @@ ifneq (,$(filter $(CONFIGURED_ARCH), armhf arm64))
 
     DOCKER_MULTIARCH_CHECK := docker inspect --type image multiarch/qemu-user-static:register &> /dev/null || (echo "multiarch docker not found ..."; docker run --rm --privileged multiarch/qemu-user-static:register --reset --credential yes)
 
-    DOCKER_SERVICE_MULTIARCH_CHECK := docker -H unix:///var/run/march/docker.sock info &> /dev/null || (echo "Docker march service not running..."; sudo rm -fr /var/run/march/docker*; (sudo $(SONIC_NATIVE_DOCKERD_FOR_MUTLIARCH) &) &>/dev/null ; sleep 1; sudo $(SONIC_USERFACL_DOCKERD_FOR_MUTLIARCH);)
+    DOCKER_SERVICE_MULTIARCH_CHECK := sudo docker -H unix:///var/run/march/docker.sock info &> /dev/null || (echo "Docker march service not running..."; sudo rm -fr /var/run/march/*; (sudo $(SONIC_NATIVE_DOCKERD_FOR_MUTLIARCH) &) &>/dev/null ; sleep 1; sudo $(SONIC_USERFACL_DOCKERD_FOR_MUTLIARCH);)
```
Comment (lguohan) [inline Makefile.work]: we are not adding sudo in the Makefile.work, need to be consistent, check line 127.

File: platform/marvell-arm64/linux/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+LINUX_KERNEL_MRVL_URL = https://github.com/Marvell-switching/sonic-marvell-binaries/raw/master/arm64/kernel/$(LINUX_KERNEL)
```
Comment (lguohan) [inline platform/marvell-arm64/linux/Makefile]: we'd like the kernel to have source code and incorporated into sonic-linux-kernel repo.

File: Makefile.work (hunk 1 above)
Comment (antony-rheneus) [reply Makefile.work]: Done.  Removed sudo.

File: platform/marvell-arm64/linux/Makefile (hunk 2 above)
Comment (antony-rheneus) [reply platform/marvell-arm64/linux/Makefile]: Ok will work on doing changes sonic-linux-kernel

**Pull request merged**